code/controllers/subsystem/ai.dm

```diff
@@ -0,0 +1,39 @@
+//A subsystem that determines what the AI mobs do; currently handles movement
+
+SUBSYSTEM_DEF(ai)
+	name = "AI Controller"
+	wait = 5
+	var/list/aidatums = list()
+	var/list/current_run
+
+	//Settings for the AI to obey
+	var/is_pacifist = FALSE //Will also ignore zones with any level of danger alongside no slashing
```

Comment:
these should be per component
if you want to toggle them globally use a global signal

Reply:
Removed until later on, will be using a potential per 'faction' system